File: src/NumericalPIDOutput.cpp
#include "NumericalPIDOutput.h"

double NumericalPIDOutput::Get() {
    return num;
}

NumericalPIDOutput::NumericalPIDOutput() : PIDOutput() {

}

void NumericalPIDOutput::PIDWrite(double output) {
    this->num = output;
}

File: src/main/java/frc/team2410/robot/NumericalPIDOutput.java
package frc.team2410.robot;

import edu.wpi.first.wpilibj.PIDOutput;


public class NumericalPIDOutput implements PIDOutput
{
	private double num;
	public double get()
	{
		return num;
	}

	public void pidWrite(double output)
	{
		this.num = output;
	}
}
